src/pools/cluster/fixed.ts

```diff
@@ -76,13 +76,6 @@ export class FixedClusterPool<
     worker.on('message', listener)
   }
 
-  protected unregisterWorkerMessageListener<Message extends Data | Response> (
-    worker: Worker,
-    listener: (message: MessageValue<Message>) => void
-  ): void {
-    worker.removeListener('message', listener)
```

Comment:
Do we never remove the listener?

Reply:
We do not need to remove the listener, we now have 1 listener for all the messages and all the workers.

Reply:
In other words, is one global listener for any message from any worker

Reply:
Ah, but we don't want to clean up the `listener` but the reference/attachment to the worker

Reply:
I think is not mandatory when a worker is destroyed the listener instance is no more attached.
To be more accurate we have one listener per worker, so when the worker got deleted bring with it also all the listeners ( i.e. we do not remove attached errorHandlers,onlinHandlers, and so on )

Reply:
@jerome-benoit  to share thoughts on that

Reply:
If worker removal is properly implemented in node.js for cluster and worker-threads, that's indeed should not be necessary to do the cleanup in poolifier.

Reply:
Yes correct, so for now I guess we are ok with this

Reply:
Please @jerome-benoit  let me know if you have any other comment